src/framework/standard/structures/mod.rs

```diff
@@ -173,6 +173,8 @@ pub struct HelpOptions {
     pub guild_only_text: &'static str,
     /// Text labeling a command's names of checks.
```

Comment:
Unrelated to your pull request, but while are at it, would be super kind if you could change this too!
```suggestion
    /// Text labelling a command's names of checks.
```